zone/zonedb.h

```diff
@@ -552,7 +552,7 @@ class ZoneDatabase : public SharedDatabase {
 	uint32		NPCSpawnDB(uint8 command, const std::string& zone, uint32 instance_version, Client *c, NPC* n = 0, uint32 extra = 0); // 0 = Create 1 = Add; 2 = Update; 3 = Remove; 4 = Delete
 	uint32		CreateNewNPCCommand(const std::string& zone, uint32 instance_version, Client* c, NPC* n, uint32 extra);
 	uint32		AddNewNPCSpawnGroupCommand(const std::string& zone, uint32 instance_version, Client* c, NPC* n, uint32 in_respawn_time);
-	uint32		DeleteSpawnLeaveInNPCTypeTable(const std::string& zone, Client* c, NPC* n);
+	uint32		DeleteSpawnLeaveInNPCTypeTable(const std::string& zone, Client* c, NPC* n, uint32 remove_spawngroup_flag);
```

Comment:
Should probably rename this to `remove_spawngroup_id` instead since this isn't _really_ a flag.

Reply:
Resolved and tested in new [commit](https://github.com/EQEmu/Server/pull/4311/commits/6524e60ba154786a0f7a5727e0717644fbd6a1d3)